src/test/lib/scheduler/operator_task_test.cpp

```diff
@@ -18,13 +18,27 @@
 #include "scheduler/node_queue_scheduler.hpp"
 #include "scheduler/operator_task.hpp"
 
+namespace {
+
+using namespace hyrise;  // NOLINT(build/namespaces)
+
+void successors_equal(auto& lhs, const std::vector<std::shared_ptr<AbstractTask>>& rhs) {
+  EXPECT_EQ(lhs.size(), rhs.size());
```

Comment:
```suggestion
  ASSERT_EQ(lhs.size(), rhs.size());
```
because the following code could segfault if executed?